assets/js/components/notifications/util.js

```diff
@@ -105,7 +105,9 @@ const removeDismissed = ( notifications ) => {
  * Remove displayed wins set to show once.
  * We display 1 win at a time so we have sometihng new for the user each time.
  *
- * @param {Array} notifications
+ * @param {Array} wins
```

Comment:
@tofumatt @aaemnnosttv I am not sure what actually a `win` is? Is that like a notification accepted or something

Reply:
Wins are basically notifications (including all errors), but "Publisher Wins" are things like increased pageviews, or traffic that Site Kit will let you know about.